res/values-de/strings.xml

```diff
@@ -0,0 +1,386 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">OctoDroid</string>
+
+    <!-- Common -->
+    <string name="record_not_found">%1$s wurde nicht gefunden</string>
+    <string name="loading_msg">Lade\u2026</string>
+    <string name="saving_msg">Speichere\u2026</string>
+    <string name="deleting_msg">Lösche\u2026</string>
+    <string name="closing_msg">Schließe\u2026</string>
+    <string name="opening_msg">Öffne\u2026</string>
+    <string name="merging_msg">Füge zusammen\u2026</string>
+    <string name="authenticating">Authentifiziere\u2026</string>
```

Comment:
Ich glaube, eine andere Formulierung würde mir hier besser gefallen ... sowas wie 'Wird geladen...', 'Authentifizierung läuft...' usw. Die englische Partizipform lässt sich halt nicht übermäßig schön in's Deutsche übertragen.